README.md

```diff
@@ -98,6 +98,7 @@ To contribute, see [CONTRIBUTING.md](./CONTRIBUTING.md)
 - [redis-viewer](https://github.com/SaltFishPr/redis-viewer) - A Redis database browser.
 - [redis_tui](https://github.com/mat2cc/redis_tui) - A Redis database browser.
 - [schemas](https://github.com/dhth/schemas) - Lets you inspect postgres schemas in the terminal.
+- [skeleton](https://github.com/termkit/skeleton): a multi-tab (TUI) library based on Bubble Tea framework.
```

Comment:
```suggestion
- [skeleton](https://github.com/termkit/skeleton) - A TUI multi-tab library based on Bubble Tea framework.
```

Reply:
The CI also fails due to this reason `101:50  List item link and description must be separated with a dash`

Reply:
Thank you for review 🍀